gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/DataNode.java

```diff
@@ -37,7 +36,17 @@
    * @return the attributes of a {@link DataNode}. It also includes properties for resolving a {@link org.apache.gobblin.runtime.api.JobTemplate}
    * e.g. "source.fs.uri" for an HDFS node, "jdbc.publisher.url" for JDBC node.
    */
-  Config getProps();
+  Config getRawConfig();
+
+  /**
+   * @return the attributes of a {@link DataNode} as a source of a {@link FlowEdge}.
+   */
+  Config getSrcConfig();
```

Comment:
I am not sure why we need to have both Src and Dst for a single DataNode, should it be either of them and relationship should be determined by the edge?

Reply:
See my response to @zxcware's similar comment. We do not need this distinction and have made the change.